relay/api/exchange/resources.py

```diff
@@ -55,7 +55,36 @@ def get(self, args):
         }
 
 
+class OrderDetail(Resource):
+
+    def __init__(self, trustlines: TrustlinesRelay) -> None:
+        self.trustlines = trustlines
+
+    args = {
+        'includeFilled': webfields.Boolean(required=False, missing=False),
+        'includeCancelled': webfields.Boolean(required=False, missing=False),
+        'includeUnavailable': webfields.Boolean(required=False, missing=False)
+    }
+
+    @use_args(args)
+    def get(self, args, order_hash: str):
+        order_orm = self.trustlines.orderbook.get_order_by_hash(bytes.fromhex(order_hash[2:]))
+
+        if order_orm is None:
+            abort(422, message='Order does not exist')
+
+        order = order_as_dict(order_orm)
+        if (args['includeFilled']):
+            order['filledTakerAmount'] = self.trustlines.orderbook.get_filled_amount(order_orm)
+        if (args['includeCancelled']):
+            order['cancelledTakerAmount'] = self.trustlines.orderbook.get_cancelled_amount(order_orm)
```

Comment:
Those are calls to the ethereum node and are to slow. We should query that from our database. I thought it is there already https://github.com/trustlines-network/relay/blob/33bde27ea4eec43c4d89c1473d3fb526d753b267/relay/exchange/exchange_db.py#L69. Otherwise please add there what is missing.

Reply:
Ah okay, atm you only get the unavailable amounts. If you need them separate, you should add that place https://github.com/trustlines-network/relay/blob/33bde27ea4eec43c4d89c1473d3fb526d753b267/relay/exchange/exchange_db.py#L46